Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
Something that uses Perl as opposed to providing an interface to itself from Perl …